src/base/static/components/organisms/tag-bar.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+import { connect } from "react-redux";
+
+import Tag from "../molecules/tag";
+import TagEditor from "../molecules/tag-editor";
+
+import {
+  getAllTagsForDataset,
+  getTagDisplayName,
+  getColorForTag,
+  getTagFromUrl,
+} from "../../state/ducks/datasets";
+
+const TagBarContainer = styled("div")({
+  borderBottom: "2px solid #ddd",
+  marginBottom: "16px",
+  paddingBottom: "8px",
+});
+
+const TagBar = props => {
+  return props.isEditModeToggled ? (
+    <TagBarContainer>
+      {props.getAllTagsForDataset(props.datasetSlug).map(tag => {
+        return (
+          <TagEditor
+            key={tag.id}
+            datasetSlug={props.datasetSlug}
+            onUpdateTagNote={props.onUpdateTagNote}
+            onDeletePlaceTag={props.onDeletePlaceTag}
+            onCreatePlaceTag={props.onCreatePlaceTag}
+            backgroundColor={tag.color}
+            displayName={props.getTagDisplayName(props.datasetSlug, tag.id)}
```

Comment:
We are already passing `datasetSlug` and `tag` into this component, and `displayName` is derived from both of those. So can we instead get the `displayName` within the TagEditor component?